app/components/page_settings_summary_component/view.rb

```diff
@@ -3,9 +3,10 @@
 module PageSettingsSummaryComponent
   class View < ViewComponent::Base
     include Rails.application.routes.url_helpers
-    def initialize(draft_question)
+    def initialize(draft_question:, long_lists_enabled:)
```

Comment:
We could default this to false to save needing to edit the calls to this component in `page_settings_summary_component_preview.rb`:

```suggestion
    def initialize(draft_question:, long_lists_enabled: false)
```

Reply:
Good idea, this will make the tidy-up easier when we remove the feature flag. Have updated